deploy/helm/charts/templates/lvm-controller.yaml

```diff
@@ -87,11 +87,12 @@ spec:
             - "--leader-election"
             - "--enable-capacity={{ .Values.storageCapacity }}"
             - "--extra-create-metadata=true"
+            - "--enable-capacity=true"
```

Comment:
Remove this, we already have this arg above.

Reply:
Done